bootstrap/lib/mix/tasks/nerves/new.ex

```diff
@@ -143,24 +150,31 @@ defmodule Mix.Tasks.Nerves.New do
   defp print_mix_info(path, extra) do
     command = ["$ cd #{path}"] ++ extra
     Mix.shell.info """
-    All set!
-      #{Enum.join(command, "\n")}
+    Your Nerves project was created successfully.
 
-    Next, pick a deployment target.
-      For example: `rpi3` for Raspberry Pi 3
-      More info on targets: https://hexdocs.pm/nerves/targets.html#content
+    You should now pick a target. See https://hexdocs.pm/nerves/targets.html#content
+    for supported targets. If your target is on the list, set `MIX_TARGET`
+    to its tag name:
 
-    To set the target you can either
+    For example, for the Raspberry Pi 3 you can either
       $ export MIX_TARGET=rpi3
-    Or prefix your commands
+    Or prefix `mix` commands like the following:
       $ MIX_TARGET=rpi3 mix firmware
 
-    Finally, Create firmware
+    If you are using an unsupported target or a custom system image,
```

Comment:
The phrase "unsupported target" seems redundant to me, given that a target not supported by a standard system would require them to use a custom system.